docs/dcp2_system_design.rst

```diff
@@ -1121,6 +1121,20 @@ Once a data release is published, the snapshots that make up the release may
 never be deleted. Superseded snapshots that were removed from the release prior
 to publishing should be deleted.
 
+All snapshots that make up a release must reside in the same `BigQuery
+location`_ so that they can be indexed simultaneously by Azul using a single
+BigQuery reservation. Reservations are location-specific. The location of a
+particular snapshot may change as long as the previous condition is satisfied
+for all releases.
+
+|nn| This would mean that snapshots can be relocated, but that all snapshots in
+a release would have to be relocated together, and to the same location. If a
```

Comment:
on the TDR front -- we pick location for data based on the data policy. So if we were hosting data that requires it to be stored in a European region int he cloud, then it breaks the model for all snapshots living in one location. We may need a discussion on this.

Reply:
Yes, if and when that happens then we need to discuss a change to the specification. My team needs this constraint to sinking more time into our around for TDR's lack of a cost effective query by primary key option (clustering).

Reply:
perfect